frontend/client/src/store/index.js

```diff
@@ -32,6 +33,8 @@ const createStore = (WrappedComponent) => {
       this.__signedInWithEmailLink = false // firebase doesn't tell us this so we need to track it ourself
       this.__authStateListener = auth.onAuthStateChanged(async (user) => {
         if (user) {
+          // make analytics accessible throughout app
+          this.__analytics = analytics
```

Comment:
Let's move this up with the other class variables, since it doesn't need data from the user login.

Also we should name it just `this.analytics`. The `__` is a convention (from python) for defining "private" properties -- but since this property is meant to be used throughout the app (like `this.store`), it shouldn't be denoted as "private".